recipes/fresnel/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "fresnel" %}
+{% set version = "0.5.0" %}
+{% set sha256 = "f5fdf02198b32088443627d732e98f0b6793bc98ad0697a37e6d75dfa5b1f663" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://glotzerlab.engin.umich.edu/Downloads/{{ name }}/{{ name }}-v{{ version }}.tar.gz
```

Comment:
@CJ-Wright If this is changed to the gitlab archive url (which i guess exists) can the bot pick up changes?

Reply:
Maybe `RawURL` will pick it up, I'm not certain. Conceivably `RawURL` would also work with the existing URL.

I would happily merge a `source` which gets the tarballs from gitlab!

Reply:
I mirror fresnel on bitbucket and github. Neither of these embed submodules in the tarballs they generate, so I release by hand at https://glotzerlab.engin.umich.edu/Downloads . The other conda-forge project I maintain, gsd, pulls tarballs from this location and the bot does detect new versions.

If you have a strong preference for gitlab generated tarballs *and* gitlab embeds submodules in its tarballs, I will consider also mirroring fresnel there.

Reply:
Yeah in that case it'd probably be fine leaving it as is.